baseframe/static/css/networkbar.css

```diff
@@ -713,3 +713,117 @@
     transition-delay: 0.25s;
   }
 }
+#hgnav {
+  background: #fff;
+  min-height: 0;
+  margin-bottom: 0;
+}
+#hgnav .page-header {
+  margin-bottom: 0;
+}
+#hgnav .page-header h1 {
+  margin-bottom: 0;
+  font-size: 16px;
+}
+#hgnav .nav {
+  margin-bottom: 0;
+}
+#hgnav .nav > li {
+  line-height: 36px;
+  color: #777;
+}
+#hgnav .nav > li > a {
+  padding: 0 15px;
+  color: #777;
```

Comment:
The button [used](https://github.com/hasgeek/baseframe/blob/hgnav-changes/baseframe/templates/baseframe/components.html#L238) is `btn-primary`, so the text needs to be in a light color, `#fff` would do.